skynet/env.py

```diff
@@ -6,14 +6,23 @@
 # general
 log_level = os.environ.get('LOG_LEVEL', 'DEBUG').strip().upper()
 supported_modules = {'summaries:dispatcher', 'summaries:executor', 'openai-api', 'streaming_whisper'}
-enabled_modules = set(os.environ.get('ENABLED_MODULES', 'summaries:dispatcher,summaries:executor').split(','))
+enabled_modules = set(
+    os.environ.get('ENABLED_MODULES', 'openai-api,summaries:dispatcher,summaries:executor').split(',')
+)
 modules = supported_modules.intersection(enabled_modules)
 
 # models
 llama_path = os.environ.get('LLAMA_PATH')
-llama_n_gpu_layers = int(os.environ.get('LLAMA_N_GPU_LAYERS', 1 if is_mac else 40))
+llama_n_gpu_layers = int(os.environ.get('LLAMA_N_GPU_LAYERS', -1 if is_mac else 40))
 llama_n_batch = int(os.environ.get('LLAMA_N_BATCH', 512))
 
+# openai api
+openai_api_key = os.environ.get('OPENAI_API_KEY')
```

Comment:
This is a good first step, but going forward we are going to need to figure this out differently. Likely by querying some vault service with some ID taken from the token.